src/components/skills/SkillItem.js

```diff
@@ -53,7 +53,7 @@ const SkillItem = ({ title, values, preview, onHiddenIconClicked, onDelete }) =>
               onClose={toggleEditSkill}
               showModal={editSkill}
               isEdit={editSkill}
-              data={editSkill ? { name: title, subSkills: values } : ''}
+              data={editSkill ? { id: id } : ''}
```

Comment:
you can simply have `editSkill ? { id } : ''`

Reply:
done [database-integrate 43bde83] database-integrate:#7 - pass just id in data param